examples/decoding/plot_ssd_spatial_filters.py

```diff
@@ -0,0 +1,139 @@
+"""
+===========================================================
+Compute Sepctro-Spatial Decomposition (SDD) spatial filters
+===========================================================
+
+In this example, we will compute spatial filters for retaining
+oscillatory brain activity and down-weighting 1/f background signals
+as proposed by :footcite:`NikulinEtAl2011`.
+The idea is to learn spatial filters that separate oscillatory dynamics
+from surrounding non-oscillatory noise based on the covariance in the
+frequency band of interest and the noise covariance absed on surrounding
+frequencies.
+"""
+# Author: Denis A. Engemann <denis.engemann@gmail.com>
+#         Victoria Peterson <victoriapeterson09@gmail.com>
+# License: BSD (3-clause)
+
+
+import matplotlib.pyplot as plt
+import mne
+from mne import Epochs
+from mne.datasets.fieldtrip_cmc import data_path
+from mne.decoding import SSD
+from mne.utils import _time_mask
```

Comment:
How about:

```import from mne.utils import _time_mask as freq_mask```

And then use ```freq_mask``` as before.

Reply:
No, this still uses a private function and is bad practice

Reply:
what would you suggest instead?

Reply:
I guess we should use _time_mask and in comment line say that this is a convenience way of extracting freq bands of interest ?

Reply:
That would still use the private function and would be ugly in my opionion as the name gives no clue on the usage.
@larsoner are you suggesting we should define our own 2 lines of example function -- potentially copying the code from _time_mask ?